applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAXAdViewWidget.java

```diff
@@ -42,6 +46,15 @@ public AppLovinMAXAdViewWidget(final String adUnitId, final MaxAdFormat adFormat
         adView.setExtraParameter( "allow_pause_auto_refresh_immediately", "true" );
 
         adView.stopAutoRefresh();
+
+        addView( adView );
```

Comment:
Can you double-check that auto-refresh does not start automatically, perhaps in the case of preloading and the publisher has not actually attached the widget to the hierarchy